resources/rest-service/cloudify/migrations/versions/396303c07e35_5_2_to_5_3.py

```diff
@@ -385,9 +386,29 @@ def _create_blueprints_labels_table():
                     unique=False)
 
 
+dl_table = sa.table(
```

Comment:
can't use `models.DeploymentLabel.__table__` because that table never has both of these columns (it had one of them in 5.2, and the other in 5.3).

Instead, declare a synthetic "table" that pretends it has both